packages/list-store/src/client-list-store.tsx

```diff
@@ -2,17 +2,18 @@
 
 import {useEffect} from 'react';
 import {useListStore} from './useListStore';
-import {SortBy} from './sort';
 import {useSearchParamsUpdate} from './useSearchParamsUpdate';
+import {defaultSortBy as packageDefaultSortBy} from './sort';
```

Comment:
The name `packageDefaultSortBy` is a bit unclear to me - what does it mean? Is there a more descriptive name?

Reply:
It's the default of the defaultSortBy :)

But I will make defaultSortBy non-optional.